src/components/hooks/useLockBodyScroll.tsx

```diff
@@ -1,6 +1,6 @@
 import React, {useLayoutEffect} from 'react'
 
-function useLockBodyScroll(props) {
+function useLockBodyScroll(props:string) {
   useLayoutEffect(() => {
     if (props === 'lock') {
```

Comment:
I suggest typing the function parameter as `action: 'lock' | 'unlock'` so you have better typings.

Reply:
Done ✅